composer.json

```diff
@@ -11,7 +11,9 @@
     ],
     "require": {
         "php": ">=5.3",
-        "symfony/framework-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.3",
+        "symfony/console": "~2.3",
+        "defuse/php-encryption": "^1.2"
```

Comment:
it should be moved to `require-dev` and `suggest` (here with description contains needed version constraints) sections